debian/tests/upgrade-all-security

```diff
@@ -53,6 +54,12 @@ chroot_exec "$chroot_dir" bash -c 'apt install -y /tmp/unattended-upgrades_*deb'
 
 
 enable_pocket "$chroot_dir" security
+chroot_exec "$chroot_dir" apt-get update
+if ! chroot_exec "$chroot_dir" apt list --upgradable 2> /dev/null | grep -q .; then
```

Comment:
Silly question - the `grep -q .` is to check if there is any output in apt list --upgradable? If so would it maybe make sense to use something like "wc" to make this more explicit? Also when I run this here with an empty upgrade set I get: 
```
LANGUAGE=en apt list --upgradable 2>/dev/null |grep  .  && echo grep-successful
Listing...
grep-successful
```
is that expected?

Reply:
This does not work, apt list --upgradable include progress output in stdout.

Reply:
Can you use apt dist-upgrade -s | grep -q ^Inst instead?

Reply:
@mvo5 @julian-klode Thanks, fixing it.

Reply:
Fixed.